integration/setup/teardown.go

```diff
@@ -44,13 +44,13 @@ func teardown(config Config) error {
 		}
 	}
 
-	if len(errors) > 0 {
-		return microerror.Mask(errors[0])
-	}
-
 	err = deleteHostPeerVPC(config)
 	if err != nil {
-		return microerror.Mask(err)
+		errors = append(errors, microerror.Mask(err))
+	}
+
+	if len(errors) > 0 {
+		return microerror.Mask(errors[0])
```

Comment:
Would it make sense to log rest of the `errors` before returning? I believe they end up being swallowed otherwise and this could lead to real issue being hidden by some warning'a'like error overriding it WDYT?

Reply:
For me it even makes sense to not return error at all in the teardown. We don't do any actions based on that anyway.

Reply:
Done.